paparazzi/src/test/java/app/cash/paparazzi/PaparazziTest.kt

```diff
@@ -108,12 +108,16 @@ class PaparazziTest {
       "onDraw time=1000 animationElapsed=0.0",
       "onAnimationStart time=2000 animationElapsed=0.0",
       "onAnimationUpdate time=2000 animationElapsed=0.0",
+      "onAnimationUpdate time=2000 animationElapsed=0.0",
       "onDraw time=2000 animationElapsed=0.0",
       "onAnimationUpdate time=2250 animationElapsed=0.25",
+      "onAnimationUpdate time=2250 animationElapsed=0.25",
```

Comment:
this is a bug; let's fix before merging this

Reply:
Yep agreed, found the culprit to my choreographer testing and updated the pr